STest/STest/Rule/Rule.hpp

```diff
@@ -60,6 +60,11 @@ namespace STest {
           const State& state //!< The system state
       ) = 0;
 
+      //! Get rule name
+      char const * getRuleNamePointer() const {
```

Comment:
This seems OK, but why not call the function `getName`?

Reply:
That makes more sense indeed, thanks!

I also removed "Pointer" from the other getters I created